e2e/404.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { test, expect, Page } from '@playwright/test';
+
+test.describe('404 Page', () => {
+  let page: Page;
+
+  test.beforeAll(async ({ browser }) => {
+    page = await browser.newPage();
+    await page.goto('/404');
+  });
+
+  test.afterAll(async () => {
+    await page.close();
+  });
+
+  test('Should have a relevant page title', async () => {
```

Comment:
```suggestion
  test('Should have a relevant page title', async ({ page }) => {
```

We can reference it directly from the function params.